src/ctapipe/calib/camera/calibrator.py

```diff
@@ -265,10 +271,12 @@ def _calibrate_dl1(self, event, tel_id):
 
             # correct non-integer remainder of the shift if given
             if self.apply_peak_time_shift.tel[tel_id] and time_shift is not None:
-                dl1.peak_time -= remaining_shift
+                dl1.peak_time = (dl1.peak_time.T - remaining_shift).T
```

Comment:
Why was this necessary? 

This also looks like it could be better with a `np.newaxis` instead of transpose and transposing back

Reply:
I cant actually remember why i used this, i think it was due to a not needed usecase  - peak_time and remaining_shift should have the same shapes so the change isnt needed. I updated it.